askama_derive/src/generator.rs

```diff
@@ -649,16 +649,18 @@ impl<'a> Generator<'a> {
                                     format!("::askama::MarkupDisplay::from(&{})", expr_buf.buf)
                                 }
                             });
-                            buf_expr.write(", ");
+                            buf_expr.writeln(",");
                         }
                     }
                 }
 
-                buf.write("write!(writer, ");
-                buf.write(&format!("{:#?}", &buf_format.buf));
-                buf.writeln(", ");
-                buf.write("\t");
+                buf.writeln("write!(");
+                buf.indent();
+                buf.writeln("writer,");
+                buf.write(&format!("{:#?},\n", &buf_format.buf));
```

Comment:
I think we can use `writeln()` here instead of including `\n` in the format string?

Reply:
It is so that `buf.start` is `false` in the following instruction.